yarn.lock

```diff
@@ -8560,7 +8560,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"jest-cli@npm:^27.5.1":
+"jest-cli@npm:^27.4.7, jest-cli@npm:^27.5.1":
```

Comment:
There was an update to jest in a merged pr, and you created a new package with the old version. Can you please update the jest version in `@lowdefy/actions-core` to `27.5.1`